orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -282,14 +287,14 @@ def AnonymiseCallback(dataset):
     tag operations through functions in pixl_dcmd module
     Returns writing anonymised dataset to disk
     """
-    orthanc.LogWarning("***Anonymising received instance***")
+    orthanc.LogWarning("Anonymising received instance")
```

Comment:
Could make this `LogInfo` as well?

Reply:
Yeah its a fair question, felt like it'd be useful to know that we've started anonymisation in case we wanted to know how long the operation takes in production, or similar things. My rule of thumb is knowing when a substantial part you care about starts and ends. Not that strongly held though if you think otherwise?